Spec lob location

andersbonielsen opened the pull request:
The specification for LOB location. Has beed added. 
README has been sligthly changed.

andersbonielsen committed:
Add files via upload

This specification is relevant for both SIARD 2.2 and SIARD 2.1

andersbonielsen committed:
LOB location specification

karinbredenberg reviewed (changes requested):
Please change the filenames so it follows what has been decided upon for the specification itself. No spaces and no dots.

andersbonielsen committed:
Rename 2024.08.01 SIARD LOB location issue.odt to 2024-08-01_SIARD_LOB_location_issue.odt

Spaces in filename replaced with underscore
Dots in filename (before suffix) replaced with dashes

andersbonielsen committed:
Rename 2024.08.01 SIARD LOB location issue.pdf to 2024-08-01_SIARD_LOB_location_issue.pdf

Spaces in filename replaced with underscore
Dots in filename (before suffix) replaced with dashes

andersbonielsen commented:
> Please change the filenames so it follows what has been decided upon for the specification itself. No spaces and no dots.
Spaces in filename replaced with underscore
Dots in filename (before suffix) replaced with dashes

andersbonielsen commented:
The issues with this branch should be solved, so that commits can be merged.
